### ISSUE TITLE
508-defect-3 [COGNITION]: Labels SHOULD be explicit in former marriages section so users understand and submit the correct data

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#13015]
## [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

While working through multiple use cases, I added a spouse to benefits, and also answered Yes to having been previously married and entered a previous marriage. When I got to the Review screen, I noticed there were implicit labels for the State and City on the previous spouse information section. I'd like to improve these labels so users ensure they are submitting accurate data. Screenshot attached below.

/cc @peggygannon for content awareness

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [x] As a user, I want to be sure the data I am submitting is correct
- [x] As a user, I want to understand what each data point represents, without seeing duplicate labels close to one another

## WCAG or Vendor Guidance (optional)

* [Error Prevention (Legal, Financial, Data): Understanding SC 3.3.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-reversible.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-28 at 4 30 47 PM](https://user-images.githubusercontent.com/934879/91617070-c4fe2e00-e94c-11ea-9276-53f5713ad745.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jason-gcio** [comment]
Let's do it.

**Comment by micahchiang** [comment]
Looking into this ticket now. An initial thought is that the field `This occurred outside the US` needs to stay in, because it's actually an entirely separate field. 

Will look into making the other two labels, `State/Country & City marriage occurred / ended` more accessible.

**Comment by micahchiang** [comment]
@1Copenut - I've tagged you as a reviewer on [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15258) that addresses the issue you've surfaced here.

**Comment by 1Copenut** [comment]
Verified in PR from previous comment. Looks good, moving to close.
